config/routes.rb

```diff
@@ -4,33 +4,34 @@
   root to: "pages#home"
 
   resources :characters, only: [:new, :show]
-  resources :journey, only: [:show], param: :slug
 
   post "/new_game", to: 'journey#create'
   delete "/end_game", to: 'journey#destroy'
   post "/continue_game", to: 'journey#continue'
+  get '/about', to: 'pages#about'
+  get '/login', to: 'sessions#new'
+  post '/login', to: 'sessions#create'
+  delete '/logout', to: 'sessions#destroy'
+  get '/dashboard', to: 'users#show'
 
-  resources :stores, only: [:show], param: :slug
   resources :trades, only: [:create]
-
   resources :categories, only: [:index, :show]
   resources :items, only: [:index, :show], param: :slug
-  resources :cart_items, only: [:create, :index, :destroy, :update]
   resources :users, only: [:new, :create, :show, :edit, :update]
-  resources :orders, only: [:index, :create, :show, :new]
-  
-  namespace :admin do
-    resources :items, only: [:index, :show, :create, :new, :update, :edit, :destroy]
-    resources :dashboard, only: [:index, :show]
-    resources :orders, only: [:index, :update]
- end
 
-  get '/about', to: 'pages#about'
-  get '/login', to: 'sessions#new'
-  post '/login', to: 'sessions#create'
-  delete '/logout', to: 'sessions#destroy'
-  get '/dashboard', to: 'users#show'
-  get '/cart', to: 'cart_items#index'
-  get '/:slug', to: 'categories#show'
-  # get '/:slug', to: redirect('/categories/%{slug}'), as: "category_name"
+  resources :stores, only: [:show], path: ":location_slug", param: :slug
+  resource :journey, only: [:show], path: ":slug", controller: 'journey' #needs to be last
```

Comment:
Line is too long. [90/80]<br>Missing space after #.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.